DomainContent/Hub/pill/scripts/swallowFXPill.js

```diff
@@ -41,12 +42,15 @@
         2, // Albedo
         3, // Normal
         5, // Metallic
-        6 // Emissive
+        6, // Emissive
+        17, // Shadow Cascade Indices
+        24, // Low Normal
+        28 // Ambient Occlusion Blurred
     ];
 
     var SWALLOW_VOLUME = 0.5;
     var EFFECT_VOLUME = 0.9;
-    var CHECK_RADIUS = 0.2; // meters
+    var CHECK_RADIUS = 0.25; // meters
```

Comment:
This is not a new change, ignore green https://github.com/highfidelity/hifi-content/blob/master/DomainContent/Hub/pill/scripts/swallowFXPill.js

Reply:
https://github.com/highfidelity/hifi-content/commit/d9ca4aca668f0b604d3676b00429854a788e03c5